test/test_events.py

```diff
@@ -319,17 +316,6 @@ def make(base, prefixes):
     return make
 
 
-def test_event_handler_event_type():
```

Comment:
It is, but a pointless one. It should have been deleted rather than just updated when we removed the `Event` superclass.